### ISSUE TITLE
[Request] when episodes missing thumbnail artwork it shows poster, instead of landscape backdrop/fanart

### ISSUE THREAD

**Issue by Nuklear92** [jurialmunkey/skin.eminence.2#90]
Is there a possibility to fix episode artwork to show landscape background backdrop/fanart instead of the show poster slapped in the middle of the landscape box when the episode doesn't have an official artwork, it looks quite bad. This happens in every type of media view that provides a landscape artwork box for Episodes (Media Views).

![image](https://user-images.githubusercontent.com/76958541/136634246-79155bd4-dbea-4bc7-9571-b37d1c6573a1.jpeg)

Eminence v2 build: 4.1.3
Kodi build: 19.2 Stable
Device: Nvidia Shield TV

**Comment by derrgoo** [comment]
Also using Fen media 2 view also looks to be all fine. 

![image](https://user-images.githubusercontent.com/63697905/136642901-60768042-8d5b-445d-9c35-aeb47ccd7b36.jpeg)

**Comment by Nuklear92** [comment]
> Is this only with specific shows? Because it appears to be working fine for me
> 
> 
> 
> Using "Your Next Episodes" list in TMDbHelper with Media Info 2 view:
> 
> ![Honeyview_screenshot00067](https://user-images.githubusercontent.com/4618719/136642148-1aeda7a0-f544-4d3d-9233-6f2fe472b0c1.jpg)
> 
> 
> 
> 
> 
> 

It works good when the Episode has an official artwork but when it's missing one like the one in the picture I shared instead of showing the background backdrop/fanart landscape it shows the Poster of the show instead in the landscape box. 

Example: Go to the TV Show: Wu-Tang Clan: An American Saga S02-E05 then go to Episode 6-7 which still Tmdb hasn't updated artwork in it. And you'll notice how instead of showing the background backdrop/fanart in the Landscape box, it instead shows the show poster. You can also try the same show I have in the screenshot I share; The Outpost S04-E12 has an official artwork which shows fine but go to the E13 and you'll see the same pattern as Wu-Tang Clan or any other show's current live Season which usually has some of the latest Episodes without official Artwork.

**Comment by derrgoo** [comment]
Possible that art is missing from tmdb or fanart.

**Comment by Nuklear92** [comment]
Oh, wait... looking at your screenshot better it's showing the Backdrop correctly. I don't understand why it doesn't in my end. I mean I don't seem to have any wrong setting for this to be happening. And it's not only in "Your Next Episodes" it's also while going directly into the TV show itself. 🤔

**Comment by Nuklear92** [comment]
Thanks! By the way I made a small clip showing the issue live. But thankfully you seem to know exactly what I mean by now. 🙂

https://streamable.com/ic9mhz

**Comment by Nuklear92** [comment]
@jurialmunkey @drinfernoo Okay I added Doc to the comment so perhaps he can double check on his end also. I installed both 4.1.5 Eminence and TMDb 4.4.7 which automatically fixed this issue to both TMDB Helper and FEN. However, when I checked addon Seren, the issue still persists even after thumbnail cleanup, database rebuild to reset all cached images/backdrops. I still don't get it, since it seems like this was a Skin wide fix you made? Now I tagged Doc because he's also using Eminence and perhaps he could double check with the Skin and Seren addon if this could be something related to that addon itself or perhaps the fix for Eminence was mainly done for the TMDb helper addon, I find it odd because it also fixed the addon FEN as you'll see in the screenshots below. 🤷‍♂️

![image](https://user-images.githubusercontent.com/76958541/136657569-3bc50711-65cb-4e1e-9e63-23b8fbf10bcf.png)
![image](https://user-images.githubusercontent.com/76958541/136657577-ab07d42a-a9fb-477f-bd06-8f9f0e74bf2b.png)
![image](https://user-images.githubusercontent.com/76958541/136657585-abfd6187-6a18-4811-a69b-32f5c454fb1c.png)
